src/lib/crypto/Kconfig

```diff
@@ -0,0 +1,41 @@
+config CRYPTO_MESSAGE_DIGEST
+	bool
+
+choice
+	prompt "Message Digest (Hash)"
+	depends on FEATURE_CRYPTO_MESSAGE_DIGEST
+	default CRYPTO_MESSAGE_DIGEST_LINUX_KCAPI if LINUX
+	help
+            Message Digest algorithms will take a byte stream and
+            compute a hash that may be used to later validate the
+            identity. Even the smallest variation of the input data
+            will have an avalanche effect that drastically change the
+            output data.
+
+            Common Message Digest algorithms are CRC32, MD5, SHA1,
```

Comment:
Associating `CRC32` with crypto is quite funny. Please make a note that `MD5` and `SHA1` should not be used unless required for interoperability. They've been broken for a while. Even `SHA256` is beginning to show its age.